Automattic/newspack-newsletters
# Update editor block paddings

### All Submissions:






### Changes proposed in this Pull Request:

Once again, this is another attempt at bringing the editor closer to the final result... the sent newsletter.

I completely removed the margins and instead I'm relying on paddings.

Let me know what you think and if it works for you!

Closes #190 

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test all the available blocks. Don't be shy!
3. You can send yourself a test or check the preview via Mailchimp.com `https://admin.mailchimp.com/campaigns/preview-content-html?id=ID_OF_YOUR_NEWSLETTER`

### Other information:
